app/mailers/rad_devise_mailer.rb

```diff
@@ -79,6 +79,8 @@ def invitation_instructions(record, token, opts = {})
   def email_changed(record, opts = {})
     @contact_log_record = record
     @contact_log_from_user = record
+    @rad_notify_on_fail = false
```

Comment:
if a user has a bad email and when they change emails and it fails sending to the old one, we don't want to notify anyone